### ISSUE TITLE
Option to display entries non-recursively

### ISSUE THREAD

**Issue by girlmaya** [authpass/authpass#302]
[Hi](https://youtube.com/watch?v=4O9vrKXUv-g) 

Using authpass 1.9.4+159 on Android.

I would appreciate an option to display the contents of a database similarly to a regular file browser, not recursively.

I appreciate the support for folders so far but I don't find much use in displaying the folders separately if opening a folder displays all contents recursively. My passwords are stored in the root of the database and some unimportant contacts and such are stored in a subfolder. I don't wish to mix these two together, it makes finding a specific password a little harder.

**Comment by hpoul** [comment]
I guess it would be possible to show a checkbox in the filter screen which currently states "recursively" to toggle between recursive and flat.. is that what you mean?

<img width="299" alt="Screen Shot 2022-06-17 at 09 02 38" src="https://user-images.githubusercontent.com/313066/174243855-3ba79f3a-9e0e-48de-8499-fd8ffcb50a69.png">
 
(ie. show only entries in the selected group, without subgroups.. so it's actually the filter which shouldn't b e non-recursive 😅 - the password entries are already listed in a flat list)

**Comment by girlmaya** [comment]
Thanks for getting back to me
I think that would work fine, though i would prefer if it listed the Groups as folders in the password list. Either way i just wish to be able to view my password list without the contents of each subfolder aswell

**Comment by dancojocaru2000** [comment]
Indeed, such a checkbox (or an entry in Preferences) would be ideal.

My usecase is that I save stuff like WiFi passwords or keys for certain services in folders dedicated to that, and I don't want those to clutter my view of passwords, I only want to see them when I'm specifically looking for them.
